Sources/Plasma/FeatureLib/pfPython/pyGUIPopUpMenu.h

```diff
@@ -127,9 +126,9 @@ class pyGUIPopUpMenu
     virtual void        SetBackSelColor( float r, float g, float b, float a );
 
     // Menu item functions
-    virtual void        AddConsoleCmdItem( const std::wstring& name, const char *consoleCmd );
-    virtual void        AddNotifyItem( const std::wstring& name );
-    virtual void        AddSubMenuItem( const std::wstring& name, pyGUIPopUpMenu &subMenu );
+    virtual void        AddConsoleCmdItem(const ST::string& name, const ST::string& consoleCmd);
+    virtual void        AddNotifyItem(const ST::string& name);
+    virtual void        AddSubMenuItem(const ST::string& name, pyGUIPopUpMenu &subMenu);
```

Comment:
While we're here, let's remove the redundant `virtual`...

```suggestion
    void        AddConsoleCmdItem(const ST::string& name, const ST::string& consoleCmd);
    void        AddNotifyItem(const ST::string& name);
    void        AddSubMenuItem(const ST::string& name, pyGUIPopUpMenu &subMenu);
```

Reply:
pfPython seems to have a lot of unneeded `virtual`s - it might be better to remove them all at once in a separate PR...